File: close10/close10.cs
using System;

namespace Testing_Exercises
{
    class MainClass
    {
        public static void Main(string[] args)
        {
            Console.WriteLine(close10(8, 12));
        }
        public static int close10(int a, int b)
        {
            if ((Math.Abs(a - 10)) == (Math.Abs(b - 10)))
            {
                return 0;
            }
            else
            {
                if (Math.Abs(a - 10) > (Math.Abs(b - 10)))
                {
                    return b;
                }
                return a;
            }
        }
    }
}

File: close10/close10.js
function close10(a,b){
    if ((Math.abs(a - 10)) == (Math.abs(b - 10))){
         console.log(0);
    }
    else {
        if (Math.abs(a - 10) > (Math.abs(b - 10))){
            console.log(b);
        }else{
            console.log(a);
        }
    }
}

close10(2,19);
